templates/nomad.systemd.erb

```diff
@@ -1,18 +1,16 @@
 [Unit]
-Description=Nomad Agent
-Wants=basic.target
-After=basic.target network.target
+Description=Nomad agent
+Requires=network-online.target
+After=network-online.target
 
 [Service]
-User=<%= scope.lookupvar('nomad::user') %>
```

Comment:
Hi 
I am also not that good in systemd but when there is credentials about user and group systemd service do not work as well.
i have tried with user and group the same script . does not worked in my case.
Regards
Karen